tests/common.py

```diff
@@ -7,24 +7,24 @@
 
 import git
 
+from oca_port.utils.misc import make_gh_info
 
-class CommonCase(unittest.TestCase):
 
-    _settings = {
-        "branch1": "15.0",
-        "branch2": "16.0",
-        "branch3": "17.0",
-        "addon": "my_module",
-        "from_org": None,
-        "from_remote": None,  # We're testing locally without any remote
```

Comment:
Not setting a remote in the tests was a quick-win to not trigger any call on GH API.
We'll have to mock GH API call if we set one.

Reply:
I'll fix this but sooner or later we'll have to mock :)